projects/components/src/header/header-content/header-content-secondary.directive.ts

```diff
@@ -0,0 +1,8 @@
+import { Directive, TemplateRef } from '@angular/core';
+
+@Directive({
+  selector: '[htHeaderContentSecondary]'
+})
+export class HeaderContentSecondaryDirective {
```

Comment:
```
@Component({
  selector: 'ht-panel-body',
  template: CONTENT_HOLDER_TEMPLATE,
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class PanelBodyComponent extends ContentHolder {}

```

Reply:
as we talked about, i went with the structural directive approach